geode-pulse/src/test/java/org/apache/geode/tools/pulse/tests/rules/ServerRule.java

```diff
@@ -72,6 +62,41 @@ protected void after() {
     }
   }
 
+  private void startServer() {
+    try {
+      server = null;
+      ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+      String jmxPropertiesFile = classLoader.getResource("test.properties").getPath();
+      int jmxPort = AvailablePort.getRandomAvailablePort(AvailablePort.SOCKET);
+      System.setProperty(PulseConstants.SYSTEM_PROPERTY_PULSE_PORT, Integer.toString(jmxPort));
+      server = Server.createServer(jmxPort, jmxPropertiesFile, jsonAuthFile);
+      server.start();
+    } catch (Exception e) {
+      throw new Error(e);
+    }
+  }
+
+  private void startJetty() throws Throwable {
+    try {
+      System.setProperty(PulseConstants.SYSTEM_PROPERTY_PULSE_HOST, LOCALHOST);
+      System.setProperty(PulseConstants.SYSTEM_PROPERTY_PULSE_EMBEDDED,
+          String.valueOf(Boolean.TRUE));
+
+      int httpPort = AvailablePort.getRandomAvailablePort(AvailablePort.SOCKET);
+      jetty = JettyHelper.initJetty(LOCALHOST, httpPort, new SSLConfig());
+      JettyHelper.addWebApplication(jetty, PULSE_CONTEXT, getPulseWarPath(), null, null);
+      pulseURL = "http://" + LOCALHOST + ":" + httpPort + PULSE_CONTEXT;
+      System.out.println("Pulse started at " + pulseURL);
+
+      if (jetty != null) {
+        jetty.start();
+      }
+    } catch (Exception e) {
+      throw new Error(e);
+    }
+  }
+
+  // Until we straighten out the Server Code properly, this should be commented out.
```

Comment:
Not sure I understand this comment.  What should be commented out?  I don't see anything commented out

Reply:
cleaned out.